_data/accessibility-tests/search.yml

```diff
@@ -0,0 +1,167 @@
+title: Search
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey the search button's meaning.
+    summary_additional: When using the search button, "search" text or a magnifying glass icon convey the button's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Search button background and text meet color contrast requirements. 
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search text button, the contrast between background and lettering is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+    wcag_criterion: 2.4.5
```

Comment:
@alex-hull and @amycole501 - is "Multiple ways" the correct WCAG criterion for this test? It seems off to me, so I thought I should flag it.

Reply:
Good catch, Sarah! Yes, that will need to be changed to 1.4.3: minimum contrast. @mahoneycm, could we change the second check under general to match as such?

Reply:
Great catch! Looks like the wcag criterion from the "multiple ways to navigate the site" got moved oddly when I merged my suggested change earlier

Reply:
Great catch! Looks like the wcag criterion from the "multiple ways to navigate the site" got moved oddly when I merged my suggested change earlier